DigitalLearningSolutions.Web/Controllers/SuperAdmin/Users/SuperAdminUserSetPasswordController.cs

```diff
@@ -47,13 +47,15 @@ public IActionResult Index(int userId, DlsSubApplication dlsSubApplication)
                 model.Page = Convert.ToInt16(TempData["Page"]);
             }
             model.userId = userId;
+            TempData.Keep();
```

Comment:
We should add the username to the formData (and submit from a hidden field) rather than using a cookie to store it, I think?